### PR TITLE
Add usage examples in doc comments

### PR DESCRIPTION
**Pull request by kirtangajjar**

### CODE REVIEW

File: src/WordPress.php [hunk 1]
```diff
@@ -164,6 +164,24 @@ public function __construct() {
...
+	 *
+	 *     # Create WordPress multisite subdom site
+	 *     $ ee site create example.com --type=wp --mu=subdom
+	 *
+	 *     # Create WordPress site with ssl from letsencrypt
+	 *     $ ee site create example.com --type=wp --ssl=le
+	 *
+	 *     # Create WordPress site with wildcard ssl
+	 *     $ ee site create example.com --type=wp --ssl=le --wildcard
+	 *
```
Comment (mrrobot47) [inline src/WordPress.php]: Add example for remote database installation. It is a very important use case.

File: src/WordPress.php [hunk 2]
```diff
@@ -637,6 +660,12 @@ private function create_site_db_entry() {
 	 *
 	 * [--db]
 	 * : Restart db container of site.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Restart all containers of site
+	 *     $ ee site restart example.com
+	 *
```
Comment (mrrobot47) [inline src/WordPress.php]: Add example for restarting particular container.

File: src/WordPress.php [hunk 3]
```diff
@@ -658,6 +687,12 @@ public function restart( $args, $assoc_args, $whitelisted_containers = [] ) {
 	 *
 	 * [--php]
 	 * : Reload php container of site.
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Reload all containers of site
+	 *     $ ee site reload example.com
+	 *
```
Comment (mrrobot47) [inline src/WordPress.php]: Add example for reloading particular container.

File: src/WordPress.php (hunk 1 above)
Comment (mrrobot47) [reply src/WordPress.php]: Also add examples with `--admin-email`, `--admin-user`, `--admin-password` parameters, and other `wp-cli` flags that we are accepting for `wp core download` `wp config create` and `wp core install`

Comment (mrrobot47) [reply src/WordPress.php]: Very important to have an example with `--locale` flag. It is a global flag and can be passed during WordPress site creation.

**Comment by kirtangajjar** [comment]
@mrrobot47 Done.

**Pull request merged**